src/test/java/htsjdk/samtools/cram/RANSInteropTestUtils.java

```diff
@@ -0,0 +1,43 @@
+package htsjdk.samtools.cram;
+
+import org.testng.SkipException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Interop test data is kept in a separate repository, currently at https://github.com/samtools/htscodecs
+ * so it can be shared across htslib/samtools/htsjdk.
+ */
+public class RANSInteropTestUtils {
```

Comment:
Unlike the actual RANS interop tests, where we agreed to change the name from from `CRAMCorpusTest` to `RANSInteropTest`, I think we intended for this file to not have "RANS" in the title, since there is nothing RANS-specific  here  - its just general utils for interop tests. So maybe `CRAMInteropTestUtils` or something.

Reply:
Also, for the future, it's better to rename (`git mv ...`) rather than delete&create, since it preserves the history. (It actually looks like you did that for the test file, just not for this one). No harm done, just mentioning it for future reference.

Reply:
done